src/gs-details-page.c

```diff
@@ -64,7 +64,8 @@
 					GS_PLUGIN_REFINE_FLAGS_REQUIRE_SIZE | \
 					GS_PLUGIN_REFINE_FLAGS_REQUIRE_SIZE_DATA | \
 					GS_PLUGIN_REFINE_FLAGS_REQUIRE_URL | \
-					GS_PLUGIN_REFINE_FLAGS_REQUIRE_VERSION
+					GS_PLUGIN_REFINE_FLAGS_REQUIRE_VERSION | \
+					GS_PLUGIN_REFINE_FLAGS_INTERACTIVE
```

Comment:
I can't see how this can have any effect. There are only 3 references to this string in the project:

1. the docstring
2. the `enum {}` definition
3. this `#define`

So there is nothing which changes its behaviour based on whether this bit is set in the flags.

I did some archaeology and on `eos3.4` through `eos3.6` we had this commit 5ea98ce517 which introduced a branch based on this flag. I don't know what happened to that commit in the `eos3.7` rebase. Let's take a look. There's no point carrying 2 patches to introduce zero behaviour change.

Reply:
> I do notice, however, that the interactive flag is now passed to refine operations through GS_PLUGIN_FLAG_INTERACTIVE, which obsoletes this patch regardless of https://github.com/endlessm/gnome-software/commit/5ea98ce517c8f4d94908f37596998dc219eef949. I'll remove that.

Sorry if I’m misunderstanding the sense of things here, but `GS_PLUGIN_FLAG_INTERACTIVE` is on its way out (perhaps I should have explicitly deprecated it by now) and is being replaced by `GS_PLUGIN_REFINE_FLAGS_INTERACTIVE` (and other `GS_PLUGIN_*_FLAGS_INTERACTIVE` flags). That’s because setting the interactive status on the entire plugin doesn’t make sense when the plugin may have multiple operations in flight concurrently.

So from how I’m interpreting what you’ve said here, this might have been a backwards step. Do we need to resurrect a dropped commit?

Reply:
Sorry, I was not aware of `GS_PLUGIN_FLAG_INTERACTIVE` being deprecated. I'll have a look during this sprint.

Reply:
There’s no real way you could have been aware of it, as it’s not officially deprecated, just deprecated in the long-term gnome-software plans inside my head :)